### PR TITLE
Make diet soda junkfood

### PR DESCRIPTION
**Pull request by XygenSS**
#### Summary
Content "Make diet soda junkfood and add sparkling water"

#### Purpose of change

Context: https://github.com/CleverRaven/Cataclysm-DDA/pull/70417#issuecomment-1872553010

#### Describe the solution

Diet soda is now junkfood
3 new non-junkfood "Lacroix type stuff"

#### Describe alternatives you've considered

Not adding unflavored sparkling water since 'tonic water' already exists (please comment whether this is correct)

### CODE REVIEW

File: data/json/items/comestibles/drink.json [hunk 1]
```diff
@@ -1751,11 +1755,78 @@
...
+    "price_postapoc": 35,
+    "material": [ "water" ],
+    "volume": "250 ml",
+    "charges": 1,
+    "phase": "liquid",
+    "flags": [ "EATEN_COLD" ],
+    "fun": 3
+  },
+  {
+    "type": "COMESTIBLE",
```
[the remark this answers is not in the document]
Comment (XygenSS) [reply data/json/items/comestibles/drink.json:1811]: Converting both zero soda and seltzers to use variants is a good idea but outside the scope of this PR I feel. The main purpose is to nerf and make zerosoda junkfood

[the remark this answers is not in the document]
Comment (NetSysFire) [reply data/json/items/comestibles/drink.json:1811]: I meant purely the things you added

[the remark this answers is not in the document]
Comment (XygenSS) [reply data/json/items/comestibles/drink.json:1811]: Gonna have to figure out how variants work lol. I'll get to it tomorrow

**Comment by fairyarmadillo** [comment]
Lacroix comes in cans, I've never seen it in a bottle.

**Comment by XygenSS** [comment]
> Lacroix comes in cans, I've never seen it in a bottle.

I googled "lacroix bottle" just in case and saw some being sold. Are they not sold in retail stores or something?

**Comment by NthTech** [comment]
In regards to the alternative discussing sparkling water vs tonic water, I think unflavoured sparkling water is different enough from tonic water to be its own item. Tonic water has quinine and sugar dissolved in addition to carbonation, whereas unflavoured sparkling water is just carbon dioxide injected into water. Without any added sugar, the item shouldn't give calories or decrement health, which differs from tonic water's current statistics of 25 calories and -1 healthy.

**Comment by fairyarmadillo** [comment]
> > Lacroix comes in cans, I've never seen it in a bottle.
> 
> I googled "lacroix bottle" just in case and saw some being sold. Are they not sold in retail stores or something?

I've never seen one. It's sold by the can in any grocery store I've ever been to.

I suppose the thing to do would be to adjust the itemgroups and make the cans more common and bottles less, but it's not a huge concern.

**Pull request merged**